unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Settings/Settings.cs

```diff
@@ -46,6 +53,14 @@ public Settings()
 
             LoadQualitySettings();
             LoadGeneralSettings();
+            LoadAudioSettings();
+
+            SubscribeToVirtualAudioMixerEvents();
+            audioMixer = Resources.Load<AudioMixer>("AudioMixer");
+        }
+
+        ~Settings() {
```

Comment:
```suggestion
        public void Dispose() {
```
Please use the `Dispose()` pattern instead of finalizers.